nexus-sled-agent-shared/src/inventory.rs

```diff
@@ -196,6 +197,18 @@ pub struct OmicronSledConfig {
     pub remove_mupdate_override: Option<MupdateOverrideUuid>,
 }
 
+impl Default for OmicronSledConfig {
```

Comment:
Thoughts on deriving this?

Reply:
This generated a lot of discussion. :sweat_smile: `Generation` not implementing `Default` is intentional, so I'll leave this as-is.